cmd/legacy_main.go

```diff
@@ -375,8 +375,15 @@ func Mount(newConfig *cfg.Config, bucketName, mountPoint string) (err error) {
 		// programme is running as daemon process.
 		env = append(env, fmt.Sprintf("%s=true", logger.GCSFuseInBackgroundMode))
 
+		var logFile *os.File
+		if newConfig.Logging.FilePath != "" {
```

Comment:
What happens if the log-file gets rotated by log-rotate? Will the file-handle continue to work?